### PR TITLE
Adds more cohort dates

### PR DESCRIPTION
**Pull request by nikeshashar**
@pitchinvasion  @danldb please can you double check the cohort dates. Thanks

### CODE REVIEW

**Comment by leo** [comment]
Wrong username!

File: data/intakes.yml [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
-  end_date: 16 December 2016
-
 - start_date: 07 November 2016
   end_date: 10 February 2017
 
 - precourse_start_date: 21 November 2016
   start_date: 03 January 2017
   end_date: 24 March 2017
+
+- start_date: 14 February 2017
```
Comment (pitchinvasion) [inline data/intakes.yml]: That's a tuesday is that right?

**Comment by pitchinvasion** [comment]
Lol with the username, also I think one of the dates is wrong

**Comment by leggsimon** [comment]
:shipit:

**Comment by pitchinvasion** [comment]
@leggsimon you sure there's no missing brackets?

File: data/intakes.yml [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
+  end_date: 28 July 2017
+
+- start_date: 19 June 2017
+  end_date: 08 September 2017
+
+- start_date: 31 July 2017
+  end_date: 20 October 2017
+
+- start_date: 11 September 2017
+  end_date: 01 December 2017
```
Comment (leggsimon) [inline data/intakes.yml]: Probably want some }}}}}}}}}}}}}}}}}} here mate

**Comment by pitchinvasion** [comment]
@leggsimon shouldn't you be fixing your broken build? https://github.com/Financial-Times/n-card/pull/161

**Comment by leggsimon** [comment]
Yeah! I don't know why it's not working!! I just keep rebuilding it and hoping for the best _fingers crossed_

**Comment by pitchinvasion** [comment]
we taught you well

**Comment by nikeshashar** [comment]
@pitchinvasion we good to go?

**Comment by pitchinvasion** [comment]
:shipit:

**Pull request merged**